deploy/osps/default/osp-rockylinux.yaml

```diff
@@ -514,6 +514,14 @@ spec:
 
               {{- template "configureProxyScript" }}
 
+              DEFAULT_IFC_NAME=$(ip -o route get 1  | grep -oP "dev \K\S+")
+
+              # enable DHCPv6 on the default interface
+              echo NETWORKING_IPV6=yes >> /etc/sysconfig/network
+              echo IPV6INIT=yes >> /etc/sysconfig/network-scripts/ifcfg-$DEFAULT_IFC_NAME
+              echo DHCPV6C=yes >> /etc/sysconfig/network-scripts/ifcfg-$DEFAULT_IFC_NAME
+              ifdown $DEFAULT_IFC_NAME && ifup $DEFAULT_IFC_NAME
+
```

Comment:
This should be configured under `/opt/bin/setup_net_env.sh` https://github.com/kubermatic/operating-system-manager/blob/master/deploy/osps/default/osp-rockylinux.yaml#L636

Reply:
I was just following how you did it for RHEL https://github.com/kubermatic/operating-system-manager/blob/8f4c16096d108b2bc443eff26599ad98699edcc3/deploy/osps/default/osp-rhel.yaml#L515

Reply:
Okay, I've checked it again and it makes sense to add this here. `/opt/bin/setup_net_env.sh` is basically just used for determining the node IP and exposing its value as an environment variable.